apps/backend/src/modules/users/users.ts

```diff
@@ -1,12 +1,14 @@
+import { encrypt } from "~/libs/modules/encrypt/encrypt.js";
 import { logger } from "~/libs/modules/logger/logger.js";
 
 import { UserController } from "./user.controller.js";
 import { UserModel } from "./user.model.js";
 import { UserRepository } from "./user.repository.js";
 import { UserService } from "./user.service.js";
+import { UserDetailsModel } from "./user-details.model.js";
 
-const userRepository = new UserRepository(UserModel);
-const userService = new UserService(userRepository);
+const userRepository = new UserRepository(UserModel, UserDetailsModel);
```

Comment:
```suggestion
const userRepository = new UserRepository(UserModell);
... somewhere in other bundle
... class UserDetailsEntity
... const userDetailsRepository = new UserDetailsRepository(UserDetailsModel)
```

Possible you need to create separate bundle "~/bundles/user-details
However, it's better to discuss this  before proceeding with such changes to avoid unnecessary rollback.

Reply:
No need to create UserDetailsEntity, UserDetailsRepository... This all will be handled within `user` module